images/scripts/opensuse-tumbleweed.setup

```diff
@@ -145,6 +145,18 @@ useradd -c Builder -G mock builder
 
 su builder -c "sudo mock --verbose -i $(/var/lib/testvm/build-deps.sh "${ID} ${VERSION_ID}")"
 
+# Setup redis
+sudo cp /etc/redis/default.conf.example /etc/redis/redis.conf
+sudo chown redis:redis /etc/redis/redis.conf
+
+mkdir -p /etc/systemd/system/redis.target.d
+cat > /etc/systemd/system/redis.target.d/70-redis-targets.conf <<EOF
+[Unit]
+Wants=redis@redis.service
+EOF
+
+systemctl enable --now redis.target
```

Comment:
I'd advise against that. We don't enable it in Fedora/RHEL images by default, and even disable it in Debian. Most tests don't need it, and Cockpit's metrics page starts it if/when necessary.